### ISSUE TITLE
Chocolatey not installing

### ISSUE THREAD

**Issue by Maitschl** [ChrisTitusTech/winutil#426]
**Describe the bug**
Chocolatey will not install. Unable to install software afterwards.

**To Reproduce**
1. Launch powershell as admin
2. get the error
**Expected behavior**
1. Launch powershell as admin
2. install choco
3. install software and enjoy

**Additional context**
 Im on a debloated Win 10 iso. I used the "lite" preset with NTLITE.

**Comment by DeveloperDurp** [comment]
can you provide a screenshoot of the errors you are getting?

**Comment by Maitschl** [comment]
sure 

![image](https://user-images.githubusercontent.com/119108157/204088331-0417e31b-fecf-41ad-9bd9-47bd21d14369.png)

**Comment by DeveloperDurp** [comment]
Have you tried rebooting? I think something in windows hasn't updated yet.

**Comment by Maitschl** [comment]
yes i tried to reboot and also everything is up2date. its on my local machine and on the vm too.

**Comment by DeveloperDurp** [comment]
Interesting, can you try running a winget install manually? 

IE something like this 

`winget install brave.brave`

**Comment by Maitschl** [comment]
yeah this is working after i accept the terms of transaction.

**Comment by DeveloperDurp** [comment]
Interesting, can you try installing something through winutil one more time? Something you have not installed yet.

**Comment by Maitschl** [comment]
Okay.. i reinstalled right now Windows 10 Edu and made a new iso with ntlite. Again i used the lite Templates and now i geht the same message but im now able to Install some things. I would like to use chrome but after the download nothing happens.. its a bit strange and maybe only related to winget?

**Comment by DeveloperDurp** [comment]
Yeah, that's one of the reasons for the idea to implement other cli tools as winget can be unstable at times. Sometimes you have to run the install a couple times to get it to work in my experience.

**Comment by UnderlingWizzo** [comment]
I have the same issue.

I have installed several apps via winget and CTT tool
Upgraded apps that were out of date
Tried all 3 different methods of running the script as on GitHUB
I got it to work by going to https://christitus.com/win, copying the script into Powershell ISE and clicking run

This has consistently worked, as I removed chocolatey and repeated all steps as previously. The only one that consistently works is copying the script into Powershell ISE and clicking run. I am not sure why it does this.

Is there any way I can help troubleshoot further?
